kin/client.py

```diff
@@ -95,45 +85,53 @@ def get_config(self):
 
         return status
 
-    def get_account_balances(self, address):
+    def get_minimum_fee(self):
+        """
+        Get the current minimum fee acceptable for a tx
+        :return: The minimum fee
+        :type: int
+        """
+        params = {'order': 'desc',
+                  'limit': 1}
+        return self.horizon.ledgers(params=params)['_embedded']['records'][0]['base_fee_in_stroops']
+
+    def get_account_balance(self, address):
         """
-        Get the KIN and XLM balance of a given account
+        Get the KIN balance of a given account
         :param str address: the public address of the account to query
-        :return: a dictionary containing the balances
-        :rtype: dict
+        :return: the balance of the account
+        :rtype: float
 
-        :raises: ValueError: if the provided address has the wrong format.
+        :raises: StellarAddressInvalidError: if the provided address has the wrong format.
         :raises: :class:`KinErrors.AccountNotFoundError`: if the account does not exist.
         """
+
+        if not is_valid_address(address):
+            raise KinErrors.StellarAddressInvalidError('invalid address: {}'.format(address))
         account_data = self.get_account_data(address)
-        balances = {}
         for balance in account_data.balances:
-            if balance.asset_code == self.kin_asset.code and \
-                            balance.asset_issuer == self.kin_asset.issuer:
-                balances['KIN'] = balance.balance
-            elif balance.asset_type == 'native':
-                balances['XLM'] = balance.balance
+            # accounts will always have native asset
+            if balance.asset_type == 'native':
+                return balance.balance
 
-        return balances
-
-    def get_account_status(self, address):
+    def does_account_exists(self, address):
         """
-        Get a given account status
-        :param str address: The public address of the account to query.
-        :return: One the possible account statuses.
-        :rtype: :enum:`kin.AccountStatus`
+        Find out if a given account exists on the blockchain
+        :param str address: The kin account to query about
```

Comment:
run pycodestyle and pydocstyle, should be a line space bertween the title and additional comments in the docstring